docker-images/kafka/Dockerfile

```diff
@@ -62,6 +62,13 @@ COPY ./stunnel-scripts/ $STUNNEL_HOME
 ENV S2I_HOME=/opt/kafka/s2i
 COPY ./s2i-scripts $S2I_HOME
 
+#####
+# Add Tini
+#####
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /usr/bin/tini
```

Comment:
Should we add some checksum notification?

Reply:
Added checksum